.all-contributorsrc

```diff
@@ -0,0 +1,276 @@
+{
+  "projectName": "kottans.github.io",
+  "projectOwner": "kottans",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [
+    {
+      "login": "programulya",
+      "name": "Julia Savinkova",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/2372431?v=4",
+      "profile": "http://www.programulya.com",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "sudodoki",
+      "name": "Джон, просто Джон",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/1506905?v=4",
+      "profile": "https://github.com/sudodoki",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "suchov",
+      "name": "Artem Sychov",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/1478985?v=4",
+      "profile": "http://artemsychov.com/",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "mayosh",
+      "name": "Julia",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/3317247?v=4",
+      "profile": "https://github.com/mayosh",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "Zaknafeyn",
+      "name": "Valentyn Radchuk",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/1470686?v=4",
+      "profile": "http://cv.vradchuk.info",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "brain-geek",
+      "name": "Oleksandr Rozumii",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/307982?v=4",
+      "profile": "https://github.com/brain-geek",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "m1n0s",
```

Comment:
I see a lot of people, who did not contributed into new website. 
What the background of this PR?

Reply:
We've discussed at some point that for a lot of people who contributed to Kottans their work is not visible, because it was removed from current state of the repo. 

While that was a necessary decision, I think we still should recognize their past work. IMO this is a good place to do that. 

Kottans are a community with a long history, that should be visible 💪

Reply:
As far as I see this right now, we have core, lectors and mentors, who mentioned on a website.

Contributers into this repo are contributers into website, who spent **a lot** of personal time and effort into it. I don't see any reason why we should bring everyone here. Probably we should find a better way to do this.

Pls, make an RFC in kottans_ongoing_projects, we should discuss, how to mention everyone.

Reply:
Sure, will do. Closing this for now

Reply:
@zonzujiro most of those contributions were created when mentors were adding themselves to mentors data yaml

Reply:
@sudodoki this is exactly what we are discussing right now https://github.com/kottans/kottans.github.io/issues/161#issuecomment-591863784 :)